### ISSUE TITLE
Missing translation for key "" when using EditButton

### ISSUE THREAD

**Issue by ceracera** [marmelab/react-admin#3017]
Whenever `<EditButton />` is used there is a warning in console `Missing translation for key: ""`
There should be no such message.

It's like button's label is not properly passed. Adding `label="Edit"` prop fixes the issue.

See: [https://stackoverflow.com/questions/54707973/warning-missing-translation-for-key/55219764#55219764](https://stackoverflow.com/questions/54707973/warning-missing-translation-for-key/55219764#55219764)

**Comment by fzaninotto** [comment]
I can't reproduce the bug you describe on the react-admin examples. Please provide more context about the reproduction of the bug. The [bug report template](https://github.com/marmelab/react-admin/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md) is a good first step.

**Comment by ceracera** [comment]
What you were expecting:
`<EditButton />` shouldn't give warning messages.

What happened instead:
There is a message in console `Missing translation for key: ""` whenever `<EditButton />` is used.

Steps to reproduce:
Add `<EditButton />` without any options to `<List />`.

Environment

    React-admin version: 2.7.2
    Last version that did not exhibit the issue (if applicable):
    React version: 16.8.3
    Browser: Firefox 65.0.1

Console output
```
Warning: Missing translation for key: "" index.js:1446

e                   React
printWarning        warning.js:34
warning             warning.js:58
warn                index.js:28
t                   index.js:395
FieldTitle          FieldTitle.js:15
renderWithHooks(12) React

**Comment by fzaninotto** [comment]
Already fixed in 2.7.3 by #2922
